packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/ValidateModelConfigHandler.java

```diff
@@ -128,12 +128,19 @@ public TaskResponse onSuccess(final TaskResponse resp) {
 				props.newModelConfigName,
 				contractedModel.getDescription()
 			);
+			contractedModelConfiguration.setTemporary(true); // Config should only be accessible within the operator for now
```

Comment:
I would reword or just remove this comment, as the notions of workflow operators are decouples from that of the taskrunner, you could in theory use the task runners without any workflows and the comment will make no sense in that context.